### PR TITLE
Feat: Routing & Home page

### PR DESCRIPTION
**Pull request by Kavita539**
Features/Changes implemented in this PR :

1. Setup basic routing for the ecommerce app using React Router
2. Home Page into full fledged React components

Components created :

- Header
- Footer
- Category
- Horizontal Card

Pages Created : 

- Home Page
- Product Page
- Cart Page
- Wishlist Page

### CODE REVIEW

File: src/pages/wishlist/Wishlist.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const Wishlist = () => {
+    return (
+      <>
```
Comment (spraveenofficial) [inline src/pages/wishlist/Wishlist.jsx:5]: No, need of using fragments here.

**Review by spraveenofficial — APPROVED** [review]

**Review by Richa-rani02 — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -1,43 +1,15 @@
...
-        </div>
-      </header>
-    </div>
+      <Routes>
+        <Route path="/" element={<Home />}/>
+        <Route path="/products" element={<Products />}/>
+        <Route path="/cart" element={<Cart />}/>
+        <Route path="/wishlist" element={<Wishlist />}/>
+      </Routes>
+      // <h1>Hello</h1>
```
Comment (prathmeshjagtap) [inline src/App.js]: you can remove this

File: src/components/horizontal-card/HorizontalCard.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+const HorizontalCard = () => {
+    const [featuredProducts, setFeaturedProducts]=useState([]);
+
+    useEffect(() => {
+        (async () => {
+          try {
+            const res = await axios.get("/api/products");
+            if (res.status === 200) {
+              setFeaturedProducts(res.data.featuredProducts);
+            //   console.log(hello)
```
Comment (prathmeshjagtap) [inline src/components/horizontal-card/HorizontalCard.jsx]: remove console.log

File: public/index.html [hunk 4]
```diff
@@ -10,6 +10,9 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cleancss.netlify.app/components/cleancss.css" />
     <!--
       manifest.json provides metadata used when your web app is installed on a
```
Comment (msdeshmukh009) [inline public/index.html]: You can remove these comments now.

File: src/App.js (hunk 2 above)
Comment (msdeshmukh009) [inline src/App.js]: Unwanted comment I guess.

File: src/components/horizontal-card/HorizontalCard.jsx (hunk 3 above)
Comment (msdeshmukh009) [inline src/components/horizontal-card/HorizontalCard.jsx]: remove this comment as well

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

**Review by prathmeshjagtap — APPROVED** [review]
I liked that you have kept code clean and structured well 
also you have given CSS styles well.

Just remove the console.log 's

File: src/pages/wishlist/Wishlist.jsx (hunk 1 above)
Comment (Kavita539) [reply src/pages/wishlist/Wishlist.jsx:5]: yeah will do it later

**Pull request merged**